### PR TITLE
Refactor route sharding

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**

### CODE REVIEW

File: content/sre/networking/route-sharding.md [hunk 1]
```diff
@@ -95,4 +95,19 @@ spec:
...
+Add DNS entry for [hello-world-app.custom-domain.com](hello-world-app.custom-domain.com) that points to the router's provisioned
+loadbalancer IP in your DNS provider. 
+
+To retrieve the loadbalancer IP: 
+```shell script
+oc get svc -n openshift-ingress router-internal --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
+```
+
+Another option is to add a wildcard entry for *.custom-domain.com to loadbalancer IP. In that way you won't have to add a separate entry
+DNS entry for each route, in turn only a single DNS entry would be required per router.
```
Comment (hamza3202) [inline content/sre/networking/route-sharding.md:111]: Typo `separate entry DNS entry` should perhaps be `separate DNS entry`

Comment (ahmedwaleedmalik) [reply content/sre/networking/route-sharding.md:111]: Fixed

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-9-1`

**Review by hamza3202 — APPROVED** [review]

**Comment by stakater-user** [comment]
@ahmedwaleedmalik Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-9-2`

**Pull request merged**